### PR TITLE
fix: Table `rowSelection.columnWidth` should work

### LINKED ISSUE #16163: `rowSelection.columnWidth` is not respected
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[Edit on CodeSandbox](https://codesandbox.io/s/r0vpl302yp)

### Steps to reproduce
Create a table that has a row selection column with custom width (<Table rowSelection={{ columnWidth: 40, ... }} ... />).

### What is expected?
The width of the row selection column should be `40px`, as specified by `rowSelection.columnWidth`.

### What is actually happening?
The width of the column is the default `60px`, not the specified custom width.

| Environment | Info |
|---|---|
| antd | 3.16.3 |
| React | 16.9.0 |
| System | MacOS 10.14.4 (18E226) |
| Browser | Chrome 73.0.3683.103 (Official Build) (64-bit) |

---
Please note that `rowSelection.columnWidth` should be applied even when the row selection column is fixed.

### PR DESCRIPTION
**Pull request by zombieJ**
### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### 👻 What's the background?

fix #16163

### 💡 Solution

Move default width style from th, td to col

### 📝 Changelog

Fix Table `rowSelection.columnWidth` not work

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/table/style/index.less [hunk 1]
```diff
@@ -685,6 +665,34 @@
   &&-scroll-position-right &-fixed-right {
     box-shadow: none;
   }
+
+  // ========================== Row Selection ==========================
+  &-with-row-selection {
+    colgroup {
+      > col:first-child {
+        width: @table-selection-column-width;
+      }
```
Comment (afc163) [inline components/table/style/index.less]: 这个选择符会不会有坑，比如

image

Comment (zombieJ) [reply components/table/style/index.less]: 试了一下，的确会有问题。看来需要 `rc-table` 里配合改一下。

**Review by ztplz — APPROVED** [review]

**Comment by zombieJ** [comment]
Update the `rc-table` to add className on the col. Please help to check again.

**Review by afc163 — APPROVED** [review]

**Comment by zuoyan188** [comment]
'columnWidth | 自定义列表选择框宽度 | string\|number | 60px',the default value  seem to be lost？

**Pull request merged**